completions/env

```diff
@@ -0,0 +1,49 @@
+# bash completion for env(1)                              -*- shell-script -*-
+
+_comp_cmd_env()
+{
+    local cur prev words cword was_split comp_args
+    _comp_initialize -s -- "$@" || return
+
+    local i noargopts='!(-*|*[uCS]*)'
+    for ((i = 1; i <= cword; i++)); do
+        if [[ ${words[i]} != -* ]]; then
```

Comment:
In addition to options, I think the the assignments of the form `var=...` should also be skipped. Otherwise, the assignments are treated as the command name in `_comp_command_offset`. Also, the later `_comp_variable_assignments` is ineffective.

Reply:
Good point, done now. The later `_comp_variable_assignments` is not entirely ineffective, it will still be helpful with completing known values for variables where supported, e.g. TZ=, LANG=, and the fallback to filedir.

Added one test, but you're right, more should be added. Marking as a draft for that will get to that soonish I hope (but feel free to chime in and push more here in the meantime if you like).

Reply:
Added some more tests, lifting draft.